Makefile

```diff
@@ -193,21 +193,32 @@ TARGETS = \
 	help \
 	info \
 	clean \
-	electron-develop
+	electron-develop \
+	publish-aws
 
 ifeq ($(TARGET_PLATFORM),darwin)
 electron-installer-app-zip: release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
 electron-installer-dmg: release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).dmg
+publish-aws: release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).dmg
```

Comment:
Notice that we usually deploy multiple artifacts per platform. For example in the case of OS X, we deploy both the DMG and a ZIP. In Windows, once we integrate it here, we will deploy executables and standalone ZIPs. How would that work here? Should we add many pre-requisites to the publish rules and iterate through them?